src/lib/utilities/get-single-attribute-for-event.ts

```diff
@@ -108,6 +109,28 @@ export const shouldDisplayChildWorkflowLink = (
   return false;
 };
 
+const keysWithParentExecutionLinks = [
+  'parentWorkflowExecutionWorkflowId',
+  'parentWorkflowExecutionRunId',
+] as const;
+
+// For linking to a parent workflow
```

Comment:
This comment is probably unnecessary given the very clear name of the function. I think if we're going to add comments, we might as well use JSDoc syntax so that Intellisense picks it up, right?